shopinvader_product_media/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_product_image_relation
```

Comment:
@mmequignon It's not the right file.... 

```suggestion
from . import test_variant_media
```